validation/spec.json

```diff
@@ -60,7 +56,7 @@
 
 	"type": "object",
```

Comment:
can you move this to the top, this will make it more intuitive that this is the toplevel object. Also I think the indenting is off.

Reply:
JSON schema tends to put definitions up top, and the top-level object at the bottom. It's a de-facto standard.. but this will actually be split into multiple separate schema files for each "object" later on. Up to you. I'll fix indenting when I get back.

Reply:
ok